gdk4/src/rgba.rs

```diff
@@ -154,3 +178,16 @@ impl FromStr for RGBA {
         RGBA::parse(s)
     }
 }
+
+impl std::ops::Deref for RGBA {
```

Comment:
Why?

Reply:
I think setters only are fine, but I had code where I get the RGBA from the gtk::StyleContext & just change the alpha value of the color; a setter should be fine I guess :)